src/brain.coffee

```diff
@@ -356,20 +356,15 @@ class Brain
     argsRe = /«__call_arg__»([\s\S]*?)«\/__call_arg__»/ig
 
     giveup = 0
-    matches = {}
+    matches = []
```

Comment:
I know it was the original name, but matches seems like a poor name, especially now.  Maybe "callSignatures" or something similar?